C/Calcium/build_tarballs.jl

```diff
@@ -23,11 +23,9 @@ import Pkg.Types: VersionSpec
 
 name = "Calcium"
 upstream_version = v"0.4.1"
-build_for_julia16_or_newer = true
-version_offset = build_for_julia16_or_newer ? v"0.0.1" : v"0.0.0"
-version = VersionNumber(upstream_version.major * 100 + version_offset.major,
-                        upstream_version.minor * 100 + version_offset.minor,
-                        upstream_version.patch * 100 + version_offset.patch)
+version = VersionNumber(upstream_version.major * 100,
+                        upstream_version.minor * 100,
+                        upstream_version.patch * 100 + 1) # remove the 1 once the upstream_version is changed
```

Comment:
We had the `version_offset` to be able to deal with rebuilds of the package. Of course you can manually add/insert the `+1` (or +2, or ...); I personally find it easier to read with the `version_offset` in place... But as usual, pure taste *shrug*

Reply:
Ah, I thought it was specifically for the < 1.6 dichotomy. I will add it back.